### PR TITLE
Table: add a select option when multiple tables exist

### PR DESCRIPTION
**Pull request by ryantxu**
The table panel currently shows the first frame in the response.  This PR adds a selection option when more than one frame exist.

We should also make it easy for the table visualization to force/recommend running transformations that convert to a single table.

image

### CODE REVIEW

**Comment by torkelo** [comment]
Does it not make sense to have the select at the top instead?

File: public/app/plugins/panel/table2/TablePanel.tsx [hunk 1]
```diff
@@ -30,20 +32,71 @@ export class TablePanel extends Component<Props> {
...
+          <div className={tableStyles.table}>
+            <AutoSizer>
+              {({ width, height }) => {
+                if (height < 5) {
+                  return null;
+                }
+                height = height - 5;
+                return <div style={{ width, height }}>{this.renderTable(data.series[index], width, height)}</div>;
+              }}
+            </AutoSizer>
```
Comment (torkelo) [inline public/app/plugins/panel/table2/TablePanel.tsx:83]: Not a fan of another auto sizer here (they are a bit expensive, slow down initial rendering), we can easily calculate the height of the select and deduct it from the panel height

Comment (torkelo) [reply public/app/plugins/panel/table2/TablePanel.tsx:83]: theme.spacing.formInputHeight  (plus whatever padding/margin)

**Comment by ryantxu** [comment]
> Does it not make sense to have the select at the top instead?

I like it better at the bottom (more like a google sheet tab)... but at the top I do not need to fix the drop down/up issue!

**Comment by torkelo** [comment]
After testing it more I agree feels better at the bottom! But tricky with the hidden menu! But think I fixed that. Think @scottlepp also has had issues with overflow hidden in panel-content class. Think I found a way to remove that. 

Made some changes to panel-content, removed the overflow hidden so that panels can have menus & popovers that extend outside. Think this wont break too many external plugins. It did not break any internal angular panels that used scrollable content.

**Review by torkelo — APPROVED** [review]

**Pull request merged**